lib/nifty_services/base_update_service.rb

```diff
@@ -44,31 +45,31 @@ def update_record
     end
 
     def can_execute?
-      unless valid_record?
-        return not_found_error!(invalid_record_error_key)
-      end
+      return not_found_error!(invalid_record_error_key) unless valid_record?
 
       if validate_user? && !valid_user?
         return not_found_error!(invalid_user_error_key)
       end
 
-      return true
+      true
     end
 
     def can_update_record?
       unless user_can_update_record?
         return (valid? ? forbidden_error!(user_cant_update_error_key) : false)
       end
 
-      return true
+      true
     end
 
     def can_execute_action?
-      return can_update_record?
+      can_update_record?
     end
 
     def user_can_update_record?
-      return not_implemented_exception(__method__) unless @record.respond_to?(:user_can_update?)
+      unless @record.respond_to?(:user_can_update?)
+        return not_implemented_exception(__method__)
```

Comment:
This is OK too, I prefer the one line return clause, but that is 90 line width code, better this way to be complaint, better for reading \o/